### DISCUSSION TITLE
Laravel 11 support

### DISCUSSION THREAD

**Discussion by jackkitley** [RobertBoes/inertia-breadcrumbs#18] [Ideas]
Please could you update to support laravel 11

  Problem 1
    - Root composer.json requires robertboes/inertia-breadcrumbs 0.4.0 -> satisfiable by robertboes/inertia-breadcrumbs[0.4.0].
    - robertboes/inertia-breadcrumbs 0.4.0 requires inertiajs/inertia-laravel ^0.6.2 -> found inertiajs/inertia-laravel[v0.6.2, ..., v0.6.11] but it conflicts with your root composer.json require (^1.0).

**Comment by RobertBoes** [comment]
Laravel 11 support has been added in https://github.com/RobertBoes/inertia-breadcrumbs/releases/tag/0.5.0
